kubernetes/kubeclient.go

```diff
@@ -36,17 +36,25 @@ import (
 
 // Golang does't support struct constants
 // Default values for an installation config
+// For resources requests and limits please see https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#how-pods-with-resource-limits-are-run
+// Only either TargetCPUUtilizationPercentage or TargetCPUAverageValue should be set in defaults, same for memory. If both are set than percentage values take precedance
 var DefaultVampConfig = models.VampConfig{
-	DatabaseName:          "vamp",
-	ImageName:             "magneticio/vampkubist",
-	ImageTag:              "0.7.13",
-	Mode:                  "IN_CLUSTER",
-	AccessTokenExpiration: "10m",
-	IstioAdapterImage:     "magneticio/vampkubist-istio-adapter-dev:latest",
-	IstioInstallerImage:   "magneticio/vampistioinstaller-dev:latest",
-	MinReplicas:           int32Ptr(1),
-	MaxReplicas:           6,
-	// Default value for TargetCPUUtilizationPercentage is nil - k8s will apply default policy
+	DatabaseName:             "vamp",
+	ImageName:                "magneticio/vampkubist",
+	ImageTag:                 "0.7.13",
+	Mode:                     "IN_CLUSTER",
+	AccessTokenExpiration:    "10m",
+	IstioAdapterImage:        "magneticio/vampkubist-istio-adapter-dev:latest",
+	IstioInstallerImage:      "magneticio/vampistioinstaller-dev:latest",
+	MinReplicas:              int32Ptr(1),
+	MaxReplicas:              6,
+	RequestCPU:               resource.NewScaledQuantity(100, resource.Milli),        // 0.1 Core
+	RequestMemory:            resource.NewQuantity(256*1024*1024, resource.BinarySI), // 256 Mi
+	LimitCPU:                 nil,
```

Comment:
We can have a default LimitCPU. These can be string and we can parse them to Quantity with https://godoc.org/k8s.io/apimachinery/pkg/api/resource#ParseQuantity

Reply:
nil means that there is no limit - we'd like to have the entire 100ms period. I thought it's reasonable default value for CPU. Would you like to make it smaller than 100ms?

Reply:
1 is logical for now. It is good to enforce limits.

Reply:
Done